packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  docker,
+  findContainerByIP,
+  dockerComposeUp,
+} from "@dappnode/dockerapi";
+import { logs } from "@dappnode/logger";
+import { params } from "@dappnode/params";
+import { getDockerComposePath } from "@dappnode/utils";
+import Dockerode from "dockerode";
+
+async function disconnectConflictingContainerAndStartBind(): Promise<void> {
```

Comment:
You could use `disconnectConflictingContainerIfAny` inside this function, right?

Reply:
re used!